mkinitramfs-ll.sh

```diff
@@ -25,6 +25,7 @@ usage() {
   -c, --compressor='gzip -9'  Use 'gzip -9' compressor instead of default
```

Comment:
Yes, there is no problem with this... added -e|--extension to command line option (you can set extension=cpio.img to the configuration file, it work as well) in d695c0e.